### DISCUSSION TITLE
Retrieve images from an external in https on the nextjs.

### DISCUSSION THREAD

**Discussion by libozro** [vercel/next.js#59525] [Help]
### Summary

I have a backend api on laravel and the front in nexjs12. the images retrieved by nextjs are in http and this does not allow the images to be displayed, only when the images are retrieved with https they display on my nextjs application. My question is is there a particular configuration to do on nextjs to retrieve all the urls in https?

### Additional information

```js
<img src="http://bledardgroup.com/storage/images/categories/101-1701295129.webp">does not display the image.

with https <img src="https://bledardgroup.com/storage/images/categories/101-1701295129.webp">display the image in my browser's inspector.
```

**Comment by pybern** [comment]
What is stopping you from replacing all the `http` images to `https`? Another option would be to create a function to do the replacements for you and wrap it around your URLs.
